pageserver/src/tenant/storage_layer/delta_layer.rs

```diff
@@ -2245,6 +2247,15 @@ pub(crate) mod test {
         (k1, l1).cmp(&(k2, l2))
     }
 
+    pub(crate) fn sort_delta_value(
+        (k1, l1, v1): &(Key, Lsn, Value),
+        (k2, l2, v2): &(Key, Lsn, Value),
+    ) -> std::cmp::Ordering {
+        let order_1 = if v1.is_image() { 0 } else { 1 };
```

Comment:
bools also impl Ord. I think `let order_1 = !v1.is_image()` achieves the same.

Reply:
yep, but I would like to make it more clear :stuck_out_tongue: